science/plumed/Portfile

```diff
@@ -113,15 +111,14 @@ pre-configure {
 # plumed-devel subport
 # This subport installs the developer version
 subport plumed-devel {
-    github.setup        plumed plumed2 d1b84c82557042ecf30543820e1d20380bd33d84
-    version             2.6-20190707
-    revision            0
+    github.setup        plumed plumed2 d6627cba892750546996b752282f200a41ed9c5e
+    version             2.6-20191907
```

Comment:
Seems this is in year, day, month format? It's better to use year, month, day format. For example, a commit on Aug 1st, 2019 should be newer than a commit on Jul 19th, 2019. If you use the year, day, month format, the version number 20190108 will be "older" than 20191907.